c++/src/kj/filesystem-disk-test.c++

```diff
@@ -910,7 +910,10 @@ KJ_TEST("DiskFile holes") {
     // Copy doesn't fill in holes.
     dir->transfer(Path("copy"), WriteMode::CREATE, Path("holes"), TransferMode::COPY);
     auto copy = dir->openFile(Path("copy"));
+#ifndef __FreeBSD__
```

Comment:
Hmm, this test used to pass on FreeBSD.

What is the error you saw, exactly? (That is, what were the values of the two `spaceUsed`?)

Reply:
```
2025-01-21T19:42:58.8173466Z [ TEST ] kj/filesystem-disk-test.c++:865: DiskFile holes
2025-01-21T19:42:58.8190932Z kj/filesystem-disk-test.c++:913: failed: expected copy->stat().spaceUsed == meta.spaceUsed [1114112 == 98304]
2025-01-21T19:42:58.8194262Z stack: 
2025-01-21T19:42:58.8195254Z kj/filesystem-disk-test.c++:927: failed: expected file->stat().spaceUsed == meta.spaceUsed [131072 == 98304]
2025-01-21T19:42:58.8195917Z stack: 
2025-01-21T19:42:58.8207994Z kj/filesystem-disk-test.c++:935: failed: expected copy->stat().spaceUsed == meta.spaceUsed [2129920 == 98304]
2025-01-21T19:42:58.8210343Z stack: 
2025-01-21T19:42:58.8223328Z [ FAIL ] kj/filesystem-disk-test.c++:865: DiskFile holes (4990 μs)
```

Reply:
Wow these numbers don't make any sense.

Can we change the TODO comments to:

```
    // The spaceUsed numbers on FreeBSD don't make any sense, but nobody has the time or interest
    // to figure out why. Oh well.
```

Reply:
The comments have been changed.

Reply:
FWIW, this happens on NetBSD 10.1:
```
[ TEST ] kj/filesystem-disk-test.c++:865: DiskFile holes
kj/filesystem-disk-test.c++:913: failed: expected copy->stat().spaceUsed == meta.spaceUsed [1114112 == 98304]
stack: 
kj/filesystem-disk-test.c++:927: failed: expected file->stat().spaceUsed == meta.spaceUsed [131072 == 98304]
stack: 
kj/filesystem-disk-test.c++:935: failed: expected copy->stat().spaceUsed == meta.spaceUsed [2129920 == 98304]
stack: 
kj/filesystem-disk-test.c++:961: failed: expected file->stat().spaceUsed < meta.spaceUsed [131072 < 98304]
stack: 
[ FAIL ] kj/filesystem-disk-test.c++:865: DiskFile holes (62517 μs)
```

Reply:
What filesystem?

Reply:
> What filesystem?

FFS, I guess:
```
$ mount
/dev/dk0 on / type ffs (local)
tmpfs on /tmp type tmpfs (local)
kernfs on /kern type kernfs (local)
ptyfs on /dev/pts type ptyfs (local)
procfs on /proc type procfs (local)
tmpfs on /var/shm type tmpfs (local)
```